cgal.sh

```diff
@@ -1,12 +1,13 @@
 package: cgal
+tag: releases/CGAL-4.6.3
 version: "4.6.3"
+source: https://github.com/CGAL/cgal
```

Comment:
Unfortunately we cannot do this for cgal. The repository is simply too big. Has this changed?

Reply:
Hm, good point. I didn't notice it particularly, but the checkout of cgal in `sw/MIRROR/cgal` is 654 MiB. I can change this back to fetching the tarball tomorrow.